lib/configuration.rb

```diff
@@ -85,6 +85,7 @@ module AlaveteliConfiguration
             :USE_MAILCATCHER_IN_DEVELOPMENT => true,
             :UTILITY_SEARCH_PATH => ["/usr/bin", "/usr/local/bin"],
             :VARNISH_HOST => '',
+            :WIDGET_PAGE => false,
```

Comment:
As mentioned in https://github.com/mysociety/alaveteli/pull/2328#discussion_r28497880, I think `ENABLE_WIDGETS` would be a better name for this key.